packages/TorneloScoresheet/src/components/Toolbar/ArbiterModeNavigation.tsx

```diff
@@ -33,84 +70,135 @@ const ArbiterModeNavigation: React.FC = () => {
     [AppMode.Recording]: false,
   };
 
+  const makeNavigationOptions = (
+    activeEventTransition: (() => void) | undefined | null,
+    assignGameTransition: (() => void) | undefined | null,
+    viewHistoryTransition: (() => void) | undefined | null,
+    willLoseData: boolean,
+  ): Option[] => {
+    const voidReturn = () => {
+      return;
+    };
+    return [
+      makeNavigationOption(
+        'Select active event',
```

Comment:
There is a lot of repeat information and logic in each of these calls. Is it worth considering a wrapper function so you only have to pass in Label and Function. All other parameters are the same and logic the same in each local call to makeNavigationOption.  The reason to do this is that review and ongoing code management will be easier.

Note that you can define a local wrapper function around makeNavigationOption